### ISSUE TITLE
This script should have a non-zero exit code on error

### ISSUE THREAD

**Issue by techmunk** [alenkacz/cert-manager-verifier#2]
When this script errors, it still returns a successful `0` exit code, and still prints that everything is "READY". It should probably have a non-zero exit code, and not print out "READY". :)

```
resource test-selfsigned already deleted
resource selfsigned-cert already deleted
Deployment cert-manager READY! ヽ(•‿•)ノ
Deployment cert-manager-cainjector READY! ヽ(•‿•)ノ
Deployment cert-manager-webhook READY! ヽ(•‿•)ノ
error when waiting for certificate to be ready: Timeout reached: context deadline exceeded
```

**Comment by alenkacz** [comment]
oh that's wrong :) I'll fix this, thanks for reporting

**Comment by alenkacz** [comment]
This is fixed in 0.2.0
